### PR TITLE
Add interpolation method for scaling.

### PR DESCRIPTION
**Pull request by FurongZhang**
Signed-off-by: FurongZhang <furong.zhang@intel.com>

### CODE REVIEW

**Comment by XinfengZhang** [comment]
duplicate with #414 ?

File: va/va.h [hunk 1]
```diff
@@ -256,6 +256,13 @@ typedef int VAStatus;	/** Return status type from functions */
 #define VA_FILTER_SCALING_NL_ANAMORPHIC 0x00000300
 #define VA_FILTER_SCALING_MASK          0x00000f00
 
+/** Interpolation method for scaling */
+#define VA_FILTER_INTERPOLATION_DEFAULT                    0x00000000
```
Comment (XinfengZhang) [inline va/va.h:260]: should you add some comments here? https://github.com/intel/libva/blob/master/va/va_vpp.h#L971

Comment (FurongZhang) [reply va/va.h:260]: What kinds of comments? Here, I have document that for interpolation method.

Comment (XinfengZhang) [reply va/va.h:260]: you will use it just like https://github.com/intel/libva/blob/master/va/va_vpp.h#L971, right?

Comment (FurongZhang) [reply va/va.h:260]: @XinfengZhang yes, I agreed with you. Will add some comments in filter_flags.

Comment (FurongZhang) [reply va/va.h:260]: Thanks @XinfengZhang for the comments. Updated.

**Pull request merged**